mitiq/observable/observable.py

```diff
@@ -73,6 +77,27 @@ def qubit_indices(self) -> List[int]:
     def nqubits(self) -> int:
         return len(self.qubit_indices)
 
+    def __mul__(
+        self, other: Union["Observable", "PauliString", complex, float, int]
+    ) -> "Observable":
+        if isinstance(other, (PauliString, complex, float, int)):
+            return Observable(*[pauli * other for pauli in self._paulis])
+        elif isinstance(other, Observable):
+            all_paulis = []
+            for other_pauli in other._paulis:
+                all_paulis += [pauli * other_pauli for pauli in self._paulis]
+            new_observable = Observable(*all_paulis)
```

Comment:
this line has been covered by the unit tests, i'm not sure why it's saying it's uncovered. (same for the one before this is just the continuation of the function definition)